src/components/Search.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { TextInput } from "@baltimorecounty/dotgov-components";
+
+const Search = props => {
```

Comment:
In this case you probably don't need to create a component for this, and can just use TextInput in the ServiceList component.

Reply:
done